packages/maskbook/src/_locales/zh/messages.json

```diff
@@ -1,123 +1,123 @@
+    "connect_to": "連接到",
+    "connecting": "正在連接…",
     "contacts": "聯絡人",
-    "groups": "分組",
-    "create": "創建",
-    "create_a_persona": "創建角色",
-    "create_persona": "創建角色",
-    "creating": "創建中…",
-    "dashboard": "儀表板",
+    "groups": "群組",
+    "create": "建立",
+    "create_a_persona": "建立一個角色",
+    "create_persona": "建立角色",
+    "creating": "建立中…",
+    "dashboard": "儀錶板",
     "dashboard_tab_assets": "資產",
-    "dashboard_tab_collectibles": "收藏品",
-    "dashboard_tab_transactions": "",
-    "dashboard_backup_database_confirmation": "好，現在備份",
-    "dashboard_backup_database_hint": "創建資料庫備份文件。要經常做。",
-    "dashboard_backup_persona_hint": "您可以通過以下任一方式備份角色。",
-    "dashboard_delete_persona_confirm_hint": "您是否真的要刪除角色 {{name}} ？此操作無法還原。",
-    "dashboard_disconnect_profile_hint": "您是否真的要从角色 {{persona}} 上斷開 {{network}} 的 {{profile}}？ 此操作無法撤銷。",
-    "dashboard_import_database_hint": "導入以前的資料庫備份。",
-    "dashboard_load_failed_title": "載入儀表板失敗。",
-    "dashboard_mobile_test": "加入移動端測試",
-    "dashboard_mobile_test_link": "https://mask.io/download-links/#mobile",
-    "dashboard_bounty_list": "",
-    "dashboard_bounty_list_link": "",
-    "dashboard_paste_database_base64_hint": "在此處粘貼 Base64 文本…",
-    "dashboard_paste_database_backup_hint": "在此處粘貼資料庫備份文本…",
-    "dashboard_persona_import_dialog_hint": "您可以通過以下任一方式導入角色備份。",
-    "dashboard_restoration_successful_hint": "恢復了 {{time}} 的備份",
-    "dashboard_source_code": "程式源代碼",
-    "dashboard_source_code_link": "https://github.com/DimensionDev/Maskbook",
-    "dashboard_email_address": "info@dimension.im",
-    "dashboard_license": "GNU AGPL 3.0",
-    "dashboard_no_collectible_found": "",
-    "database_group_followers_name": "{{owner}} 的關注者",
-    "database_group_following_name": "{{owner}} 關注的人",
-    "database_group_friends_default_name": "{{owner}} 的好友",
-    "decrypted_postbox_add_recipients": "添加收件人",
-    "decrypted_postbox_decrypting": "Mask 正在解密…",
-    "decrypted_postbox_decoding": "Mask 正在解码…",
-    "decrypted_postbox_decrypting_finding_person_key": "Mask 正在搜尋對方的公鈅…",
-    "decrypted_postbox_decrypting_finding_post_key": "Mask 正在搜尋分享給您的密鑰以解密这篇内容…",
-    "decrypted_postbox_author_mismatch": "最初为 {{name}} 分享的讯息",
-    "decrypted_postbox_title": "此消息使用 Mask 解密：",
-    "dismiss": "",
-    "details": "",
+    "dashboard_tab_collectibles": "收藏",
+    "dashboard_tab_transactions": "交易",
+    "dashboard_backup_database_confirmation": "好的，開始備份",
+    "dashboard_backup_database_hint": "建立一個資料庫備份檔。 請定期備份。",
+    "dashboard_backup_persona_hint": "您可以使用下列選項備份您的角色。",
+    "dashboard_delete_persona_confirm_hint": "您確定要刪除 {{name}} 這個角色嗎? 這個操作是無法復原的。",
+    "dashboard_disconnect_profile_hint": "您是否要切斷 {{network}} 上的設定檔 {{profile}} 與角色 {{persona}} 的連結? 這個操作是無法復原的。",
+    "dashboard_import_database_hint": "自一個之前的資料庫備份還原。",
+    "dashboard_load_failed_title": "無法加載儀錶板。",
+    "dashboard_mobile_test": "加入手機版本測試",
+    "dashboard_mobile_test_link": "https://mask。io/download-links/#mobile",
```

Comment:
？？？？

Reply:
oops sorry about that
should be fixed in 0d77395e